pkg/daemon/manager/manager.go

```diff
@@ -222,3 +254,19 @@ func enumDependentServicesSyscall(hService windows.Handle, dwServiceState uint32
 	return enumDependentServicesW.Call(uintptr(hService), uintptr(dwServiceState), uintptr(unsafe.Pointer(lpServices)),
 		uintptr(cbBufSize), uintptr(unsafe.Pointer(pcbBytesNeeded)), uintptr(unsafe.Pointer(lpServicesReturned)))
 }
+
+// waitForProcessToStop waits until the process has exited
+func waitForProcessToStop(process windows.Handle) error {
+	return wait.PollImmediate(retry.WindowsAPIInterval, retry.ResourceChangeTimeout, func() (done bool, err error) {
+		var exitCode uint32
+		if err := windows.GetExitCodeProcess(process, &exitCode); err != nil {
+			// unexpected error, most likely related to permissions
+			return false, errors.Wrap(err, "error getting process exit code")
+		}
+		// STATUS_PENDING indicates the process has not exited, keep retrying.
+		if exitCode == uint32(windows.STATUS_PENDING) {
```

Comment:
Should/can we also check for `STILL_ACTIVE`? [Microsoft docs](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-getexitcodeprocess):

> If the process has not terminated and the function succeeds, the status returned is STILL_ACTIVE (a macro for STATUS_PENDING (minwinbase.h))

Reply:
> a macro for STATUS_PENDING

This implies they are the same value, `0x00000103`